### ISSUE TITLE
awesome project

### ISSUE THREAD

**Issue by xiangwan** [nylas/electron-RxDB#1]
👍

**Comment by na-ji** [comment]
I agree with @xiangwan! I just migrated from pouchdb to RxDB and now everything is so fast in my app. I used to wait around 500ms to fetch a thousand records, now it takes less than 50ms. Great job @bengotow and the nylas team!

**Comment by Ramblurr** [comment]
Does this project completely replace pouchdb under the hood?

I would like to use electron-rxdb to persist data in a desktop application, but still replicate to another couchdb/pouch server.

**Comment by xiangwan** [comment]
@Ramblurr try https://github.com/pubkey/rxdb
